CMake/deps/rippled.cmake

```diff
@@ -1,5 +1,5 @@
-set(RIPPLED_REPO "https://github.com/ripple/rippled.git")
-set(RIPPLED_BRANCH "1.9.2")
+set(RIPPLED_REPO "https://github.com/gregtatcam/rippled.git")
+set(RIPPLED_BRANCH "amm-core-functionality")
```

Comment:
We can't merge this. Need a plan

Reply:
Yup, I just added this file. So that it's easy for you test. This file is not needed in the merge.

Reply:
Similarly, the config file is also not needed in the merge.

Reply:
> Yup, I just added this file so that it's easy for you to test. This file is not needed in the merge.

The problem is that it does not build without this as we are using stuff that is added to rippled in the AMM pr.. yet we can't merge this to develop. We will need to figure out how we go about it. No worries :+1: